templates/info.html

```diff
@@ -31,7 +31,7 @@ <h4>Difficulty Level - {{idea[i]['Idea']['sub_ideas'][j]['ideas']['DifficultyLev
                 </ul>
                 <h4>Potential Mentors</h4>
                 <p class="lead font-weight-normal">
-                  {% for mentor in idea[i]['Idea']['sub_ideas'][j]['ideas']['PotentialMentors'] %}
+                  {% for mentor in idea[0]['Idea']['sub_ideas'][0]['ideas']['PotentialMentors'] %}
```

Comment:
here and later `idea[i]...` was correct. Why did you change it?

Reply:
Actually in the content.yaml file I am not again and again writing the name of the mentors. because all are same .It was simply a repetition.

Reply:
As in all the fields the name of the mentors are same , so I wrote at one place only .If you say then I write for every idea.